.gitignore

```diff
@@ -8,6 +8,9 @@ keys/
 docs/generated
 docs/README.rst
 docs/html/
+remme/protos/
 !remme/protos/__init__.py
 docker-compose/default_export
 logs/
+poetry.lock
+.pytest_cache
```

Comment:
Add new line to the end of the file. [Because of](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline). Check all files in pull request to fix this point.